internal/adapter/types/labels.go

```diff
@@ -27,4 +30,10 @@ const (
 
 	// WorkloadNameLabelKey is the key used to store the workload name in the container labels
 	WorkloadNameLabelKey = "workload.k2d.io/name"
+
+	// PersistentVolumeLabelKey is the key used to store the persistent volume name in the container labels
+	PersistentVolumeLabelKey = "persistentvolume.k2d.io/name"
```

Comment:
See if you can put all of the storage related labels under storage.k2d.io/pv, storage.k2d.io/pvc etc